app/code/Magento/Cms/Model/Wysiwyg/Images/Storage.php

```diff
@@ -22,6 +22,7 @@
  * @SuppressWarnings(PHPMD.TooManyFields)
  * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
  * @SuppressWarnings(PHPMD.CookieAndSessionMisuse)
+ * @SuppressWarnings(PHPMD.CyclomaticComplexity)
```

Comment:
Hello @engcom-Charlie ,
I tried to suppress the warning to fix the static test but not able to see its working or not?
Can you please help me out?
Thanks